packages/gatsby/src/query/__tests__/query-compiler.js

```diff
@@ -96,3 +111,1181 @@ describe(`resolveThemes`, () => {
     ).toEqual([expect.stringContaining(theme.split(`/`).join(path.sep))])
   })
 })
+
+describe(`actual compiling`, () => {
+  let schema
+  beforeAll(() => {
+    schema = buildSchema(`
```

Comment:
I wonder if we should put this schema inside a fixture folder and read from disk instead of putting it all here.